pkg/utils/helper.go

```diff
@@ -251,22 +251,13 @@ func GetCreationTimeStamp() (string, error) {
 }
 
 // WriteToCRD writes diagnostic data to CRD
-func WriteToCRD(fileName string, key string) error {
-	hostName, err := GetHostName()
-	if err != nil {
-		return err
-	}
+func WriteToCRD(jsonBytes []byte, key string, hostName string) error {
```

Comment:
not the point of this PR but why do we need this function?

Reply:
"diagnostics" values are exposed via a k8s custom resource created under the periscope api namespace (see helper.go  createCRD() and writeDiagnosticCRD())